### PR TITLE
Some changes

### PR DESCRIPTION
**Pull request by mikiquantum**

### CODE REVIEW

File: docs/getting-started/install-cent-node.md [hunk 1]
```diff
@@ -45,13 +45,41 @@ For instructions to install `geth` on MacOS, see [Instructions for MacOS](https:
...
+* 2 cores
+  * Syncing mode uses about 2-3 cores
+  * Normal mode uses about 0.2 cores
+* Allocated disk space of at least 100 Gigabyte
+  * Sync mode uses about 1 Gigabyte
+  * Full node requires about 50 Gigabyte
+* Syncing will take about 1-2 hours. This can vary depending on the resource availability of the full node(s).
+  * Optionally, you can use SSDs speed it up.
+
+Start the `geth` process, we chose running `go-ethereum` in light mode, for speed and simplicity, feel free to tailor this setup to fit your needs:
```
Comment (dnavale) [inline docs/getting-started/install-cent-node.md]: Start the `geth` process:

  ```bash
  $ geth --rinkeby --light --rpc --rpcapi db,eth,net,web3,txpool --ws \
  --wsorigins "*" --wsapi db,eth,net,web3,txpool > /tmp/geth.log 2>&1 &
   ```
This command runs `go-ethereum` in the _light_ mode for speed and simplicity. You can modify this command based on your requirements.

File: docs/getting-started/install-cent-node.md [hunk 2]
```diff
@@ -69,52 +97,35 @@ Once you have installed all the necessary packages, follow these steps to instal
   -a 8082 -p 38204
   ```
 
-  This command automatically adds the Identity keys to the `config.yaml` file.
+  This command automatically creates an Identity and keys needed and generates the `config.yaml` file required to run the node.
```
Comment (dnavale) [inline docs/getting-started/install-cent-node.md]: This command automatically creates an Identity and the key pairs. It then generates the `config.yaml` file required to run the node.

File: docs/getting-started/install-cent-node.md [hunk 3]
```diff
@@ -69,52 +97,35 @@ Once you have installed all the necessary packages, follow these steps to instal
...
   $ centrifuge run -c /PATH-TO-CONFIG-FILE/config.yaml
   ```
   Replace the `_PATH-TO-CONFIG-FILE_` with the location of the `config.yaml` file.
 
-### Installing the Ethereum Rinkeby Node
+#### Additional custom configuration
+* Configure node under NAT
 
-Rinkeby is a testnet
+If you want your node to be reachable outside of your private network you will need to manually specify the External IP where the node will be available:
```
Comment (dnavale) [inline docs/getting-started/install-cent-node.md]: If you want your node to be accessible outside your private network, you will need to manually specify the External IP of the node:

File: docs/getting-started/install-cent-node.md [hunk 4]
```diff
@@ -69,52 +97,35 @@ Once you have installed all the necessary packages, follow these steps to instal
...
-
-3. Start the `geth` process:
-
-  ```bash
-  $ geth --rinkeby --light --rpc --rpcapi db,eth,net,web3,txpool --ws \
-  --wsorigins "*" --wsapi db,eth,net,web3,txpool > /tmp/geth.log 2>&1 &
-   ```
-
-You now need to wait for about 1-2 hours depending on the resources under the P2P connected network for the local node to sync up with the Rinkeby network.
+We provide a way to your upstream systems to receive events whenever a new document has been shared with your node, and we do it through WebHooks:
```
Comment (dnavale) [inline docs/getting-started/install-cent-node.md]: To receive an event when a new document has been shared with your node, set your WebHook endpoint in the `config.yaml` file:

File: docs/getting-started/install-cent-node.md [hunk 5]
```diff
@@ -69,52 +97,35 @@ Once you have installed all the necessary packages, follow these steps to instal
...
+```yaml
+notifications:
+  endpoint: "YOUR_WEBHOOK_ENDPOINT"
+```
+You can find information about the notification payload here:
```
Comment (dnavale) [inline docs/getting-started/install-cent-node.md]: For more information, see the [Notification Payload](https://app.swaggerhub.com/apis/centrifuge.io/cent-node/0.0.1#/NotificationDummyService/Notify).

**Comment by mikiquantum** [comment]
@dnavale applied your comments.

**Comment by dnavale** [comment]
@mikiquantum, are you happy for me to merge this PR? Or, do we wait until @rdinicut and @lucasvo  have reviewed this as well? 

I have some additions for install chapter as well.. but want to make sure I include them once this is merged to avoid conflicts.

**Comment by rdinicut** [comment]
@dnavale if the build passes and at least one person has reviewed the code it is ok cu merge it. Be sure to select squash and merge not merge

**Comment by mikiquantum** [comment]
@dnavale yes lets get this in!

**Pull request merged**